descriptors/ModuleDescriptor-template.json

```diff
@@ -137,7 +147,7 @@
     },
     {
       "id": "inventory-batch",
-      "version": "0.3",
+      "version": "0.5",
```

Comment:
Why has this version changed by two increments?

Reply:
The thing is that inventory-batch.raml has v0.4. It isn't consistent with ModuleDescriptor-template.json That why I've set 0.5 version

Reply:
Was this due to a mistake in a previous change and only the RAML file was changed and not the descriptor?